components/dashboard/Sidebar.tsx

```diff
@@ -0,0 +1,83 @@
+import styles from '../../styles/components/Sidebar.module.scss'
+import {SocialLinks} from "../common/SocialLinks/SocialLinks";
+import {FaHome} from "react-icons/fa";
+import {MdAccountCircle, MdLogout} from "react-icons/md";
+import {BiCalendarEvent} from "react-icons/bi";
+import {AiFillTrophy, AiOutlineMenuUnfold} from "react-icons/ai";
+import {SidebarItem} from "./SidebarItem";
+import {useState} from "react";
+import {CgClose} from "react-icons/cg";
+
+const sidebarItems = [
+    {
+        title: "Home",
+        icon: <FaHome />,
+        path: "/dashboard"
+    },
+    {
+        title: "My Profile",
+        icon: <MdAccountCircle />,
+        path: "/user/me"
+    },
+    {
+        title: "Leaderboard",
+        icon: <AiFillTrophy />,
+        path: "/dashboard/leaderboard"
+    },
+    {
+        title: "Events",
+        icon: <BiCalendarEvent />,
+        path: "/dashboard/events"
+    },
+    {
+        title: "Logout",
+        icon: <MdLogout />,
+        path: "/idk"
+    }
+]
+
+export const Sidebar = () => {
+
+    const user = {
+        name: "Ken",
+        designation: "Member",
+        dp: "https://avatars.githubusercontent.com/u/78747188?v=4"
+    }
+
+    const [mobileSidebarOpen, setMobileSidebarOpen] = useState("false")
+
+    return (
+        <>
+            <AiOutlineMenuUnfold
```

Comment:
Need to change icon too, but we will decide this later.

Reply:
Will replace with hamburger component of navbar